### ISSUE TITLE
Make software deactivation of touchpad optional

### ISSUE THREAD

**Issue by jf99** [linuxmint/Cinnamon#5515]
My laptop computer has a button to toggle the touchpad between on and off. When I press it, Cinnamon recognizes it:

![off](https://cloud.githubusercontent.com/assets/7962377/17361473/fd048512-5970-11e6-8460-d03a278c3aa7.png)
![on](https://cloud.githubusercontent.com/assets/7962377/17361478/03fe9308-5971-11e6-9260-21ec6d77713f.png)
The problem is, my computer already enables/disables its touchpad in hardware. And sometimes (for example after waking up from standby) hardware and software get out-of-sync. This leads to situations where either Cinnamon or the hardware itself is blocking my touchpad, no matter how often I press this button to get it work again.

To solve this annoyance, I would suggest to make the software recognition of the button optional.

**Comment by icarter09** [comment]
@jf99 are you still experiencing this issue?

**Comment by jf99** [comment]
@icarter09 Yes, this does still happen with Cinnamon 4.4.3. A setting whether to ignore the hardware switch is missing.

**Comment by icarter09** [comment]
@jf99 thanks for the update.
